applications/ShapeOptimizationApplication/tests/opt_process_vertex_morphing_test/run_test.py

```diff
@@ -114,7 +114,7 @@ def __TentFunction(x):
 
     # Check against specifications
     TestCase().assertEqual(resulting_optimization_iterations, 124)
-    TestCase().assertAlmostEqual(resulting_improvement, -90.228425, 5)
+    TestCase().assertAlmostEqual(resulting_improvement, -90.2284, 4)
```

Comment:
Did this test become more sensitive because of the current changes?

Reply:
No, just the output changed.